licenses/BSL.txt

```diff
@@ -17,7 +17,7 @@ Additional Use Grant: You may make use of the Licensed Work, provided you
                      * Use it for commercial purposes with more than 500
                      monthly active client instances.
 
-Change Date:          2024-01-25
+Change Date:          2024-02-08
```

Comment:
Did `bump` do this? It's not supposed to affect the `Change Date` except if the major version changes.